lib/ExportsFieldPlugin.js

```diff
@@ -102,7 +104,17 @@ module.exports = class ExportsFieldPlugin {
 				forEachBail(
 					paths,
 					(p, callback) => {
-						const error = checkExportsFieldTarget(p);
+						const queryIndex = p.lastIndexOf("?");
```

Comment:
```suggestion
						const queryIndex = p.indexOf("?");
```

query string could contain `?` itself